tree/treeplayer/inc/ROOT/TDFInterface.hxx

```diff
@@ -1574,8 +1609,7 @@ private:
       // We share bits and pieces with snapshot. De facto this is a snapshot
       // in memory!
       TDFInternal::CheckSnapshot(sizeof...(BranchTypes), columnList.size());
-
-      std::tuple<TDFInternal::CacheColumnHolder<BranchTypes>...> colHolders;
+      std::tuple<TDFInternal::CacheColumnHolder<typename TDFDetail::TakeRealTypes<BranchTypes>::RealColl_t::value_type>...> colHolders;
```

Comment:
is `TakeRealTypes<BranchTypes>::RealColl_t::value_type` different from `TakeRealTypes<BranchTypes>::RealT_t`?